backend/aws_helpers/dynamo_db_utils/file_upload_db.py

```diff
@@ -1,152 +1,62 @@
-import json
-
-from dataclasses import dataclass
-from datetime import datetime
-
-from backend.aws_helpers.dynamo_db_utils.base_db import (
-    BaseData,
-    BaseDDBUtil,
-    enumclass,
-    changevar,
+from backend.aws_helpers.dynamo_db_utils.dynamo_db_utils import (
+    create_dynamo_item,
+    get_dynamo_item_by_id,
+    get_dynamo_items_by_gsi,
+    update_dynamo_item,
 )
-from backend.common.constants import FILE_UPLOAD_TABLE_NAME, AWS_REGION
-from boto3.dynamodb.conditions import Key
-from backend.common.utils import get_current_timestamp, get_current_unix_time
-from typing import Union
-
-PRIMARY_KEY = "s3_uri"
+import random
+from datetime import datetime
+from dataclasses import dataclass
 
-REQUIRED_KEYS = [
-    "s3_uri",
-    "uid",
-    "created",
-    "filename",
-]
+TABLE_NAME = "dlp-file-upload-table"
```

Comment:
isnt this table name already defined in dynamo db utils constants?

Reply:
@farisdurrani

Reply:
Very good point. Fixed in latest commit